resources/snakefiles/mapping.smk

```diff
@@ -119,16 +119,16 @@ rule sort_index_bam:
     input:
         aln="output/mapping/{mapper}/mapped_reads/{read_sample}_Mapped_To_{contig_sample}.bam"
     output:
-        bam=temp("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.sorted.bam"),
-        bai=temp("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.sorted.bam.bai")
+        bam=temp("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.bam"),
+        index=temp(touch("output/mapping/{mapper}/sorted_bams/{read_sample}_Mapped_To_{contig_sample}.bam.bai"))
```

Comment:
You shouldn't need a touch here at all, as this rule is creating the .bai file.

Reply:
ok, done!

Reply:
whoops, looks like I pushed that update to the "Distribute and run samples" PR.